src/libostree/ostree-autocleanups.h

```diff
@@ -59,6 +59,10 @@ G_DEFINE_AUTOPTR_CLEANUP_FUNC (OstreeSysrootUpgrader, g_object_unref)
 
 G_DEFINE_AUTO_CLEANUP_CLEAR_FUNC (OstreeRepoCommitTraverseIter, ostree_repo_commit_traverse_iter_clear)
 
+#ifdef OSTREE_ENABLE_EXPERIMENTAL_API
```

Comment:
> That seems OK, but it'd probably be worth at least a comment in configure.ac that the name of the define is effectively API.

It’s deliberate — but you’re right, I should call it out in a comment. Will add one.

I could use a different `#define` if you want. For greppability, I kept them the same, but I’m not wedded to this idea.